dev-utils/test-config.js

```diff
@@ -92,6 +92,10 @@ function getBrowserList() {
       browserName: 'chrome',
       version: '62'
```

Comment:
Maybe we should just change this to 74 instead, since their market share drops significantly after a new updates comes out!

Reply:
I started this test after one of our discuss issue on chrome 74. Let's come up with some small list that we have to run tests on. I was thinking to have 1 with least -> mid range -> current/latest. What are your thoughts?

Reply:
I think we should keep the lowest version and the highest! IMO, the benefit of having a mid decreases as new versions become available since in most cases it will automatically update and in the case of chrome new versions are quite frequent!

Reply:
Yeah sure. I will come update the list with new versions on each platform and will move the PR status accordingly. 

DT works fine in chrome 74. Probably you can link the PR to the discuss chat if you would like the user to have a look.